package.json

```
@@ -3,7 +3,7 @@
   "version": "1.2.4",
   "scripts": {
     "ng": "ng",
-    "start": "node version.js && ng serve",
+    "start": "node version.js && ng serve --port 8090",
```

```suggestion
    "start": "node version.js && ng serve",
```
Ne pas spécifier de port dans le script

---

I'm Okay with that without using specified port number, we can make it more generic (I set this port for debuging becuase running more than two angular projects simultaneously).

---

What is pushed on master is what everyone cloning the project will get, so it is important that we keep it generic.